crates/p2p/src/swarm/errors.rs

```diff
@@ -23,26 +23,37 @@ pub enum Error {
 /// Validation errors.
 #[derive(Debug, Error)]
 pub enum ValidationError {
+    /// The signature is invalid.
     #[error("Invalid signature")]
     InvalidSignature,
+    /// The message signer is not in the signer's allowlist.
     #[error("Not in signers allowlist")]
     NotInSignersAllowlist,
 }
 
+/// Errors from libp2p
```

Comment:
I don't think these are "errors from libp2p" exclusively.
Please review your docstring. We have `tokio` channel errors for example.